src/components/gds-menu/gds-menu.stories.js

```diff
@@ -35,12 +35,12 @@ export const Examples = () => html`
 
 // prettier-ignore
 export const Submenus = () => html`
-  <gds-menu>
+  <gds-menu role="menubar" aria-label="Primary navigation">
```

Comment:
Not sure if we should just use accessible-label for consistency even if the aria attribute works in this case